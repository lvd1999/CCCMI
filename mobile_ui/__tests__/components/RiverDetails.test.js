import 'react-native';
import React from 'react';
import RiverDetail from '../../components/RiverDetails';
import renderer from 'react-test-renderer';

describe.only('RiverDetail component test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<RiverDetail />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
