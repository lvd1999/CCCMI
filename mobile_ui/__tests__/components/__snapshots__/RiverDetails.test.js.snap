// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RiverDetail component test renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
      "width": "95%",
    }
  }
>
  <RCTScrollView
    contentInsetAdjustmentBehavior="automatic"
  >
    <View>
      <View
        style={
          Object {
            "marginTop": 30,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#1B262C",
              "fontSize": 30,
              "paddingBottom": 30,
            }
          }
        >
          River details
        </Text>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            name
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            Stream Liffey
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            latitude
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            53.3539
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            longitude
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            -6.3557
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            area
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            23.58
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            length
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            15.41
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            ecologicalStatus
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            Good
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            waterbodyCategory
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            River
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            localAuthority
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            Kildare County Council
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            wfdRisk
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            Review
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            protectedArea
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            Yes
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            canal
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            No
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "rgba(149,152,154,0.3)",
              "borderBottomWidth": 2,
              "marginBottom": 20,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 15,
                  "opacity": 0.6,
                },
              ]
            }
          >
            cycleRbd
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#1B262C",
                },
                Object {
                  "fontSize": 20,
                  "opacity": 0.87,
                },
              ]
            }
          >
            Eastern
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
